public/style.css

```diff
@@ -101,15 +102,33 @@ header{
     width: 130px;
     height: 40px;
   }
+
+  .action-btn:hover,
+  .edit-btn:hover,
+  .delete-btn:hover{
+    background-color: #4451ff;
+  }
+
   
   .create-btn {
     margin-top: 20px;
-    width: 130px;
-    height: 40px;
+    width: 150px;
```

Comment:
Should we make all our buttons `150px`? so that the sizes are uniform or is this to draw attention to the button?